### ISSUE TITLE
Program doesn't open

### ISSUE THREAD

**Issue by darlopvil** [xiaoyifang/goldendict#31]
Hello there,

I've always used the latest alpha version of [GoldenDict](https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Windows#150-rc2-372-gc3ff15f), but that the other day it just stopped working correctly. Honestly, I've always had problems with that version (usual crashes, for example). So i read about this goldendict fork.

After downloading it [from the releases page](https://github.com/xiaoyifang/goldendict/releases/tag/v22.2.alpha-a088b0e1) (the 6.2.3 .zip version file), uncompressed it, and then executed the .exe file, appeared the classic small window loading some dictionaries. But it stopped and didn't open the main window of the program. It also disappeared the icon from the Windows Task Bar. It seems that this fork and the original version suffer from the same issue. Do you happen to know what may be the problem?

**Comment by xiaoyifang** [comment]
need more information

**Comment by darlopvil** [comment]
like what?

This is what happens: 
I open the program, a small window appears where the dictionaries are loaded. I see the GoldenDict icon appear on the desktop toolbar. When the dictionaries finish loading, the window and the icon disappear. In other words, the program never finishes opening.

**Comment by darlopvil** [comment]
Yes, i work on Win10.

After doing what you said, the program did open, but it crashed shortly after linking my dictionary path :( Never finishes loading the dictionaries, so i guess it's the same as before.

**Comment by xiaoyifang** [comment]
when running in console ,does it give any error message.

other other ways to try in order to identify the cause:
1,move you dictionary to other path.  then open goldendict to make certain if the crash comes from dictionary of not.
2, if step 1 succeed ,add you dictionary one by one and check which dictionary cause the crash.

**Comment by xiaoyifang** [comment]
closed because has no more info.
reopen if neccessary.

**Comment by darlopvil** [comment]
I've been busy these days, sorry for not replying earlier. The problem still continues, unfortunately. 

While doing this, found that there were a couple of copies of older versions of goldendict installed in my PC. Obviously, and seeing all the mess, all remains of goldendict dictionaries files were immediately erased but only saving the dictionaries (I've 25 gbs of dicts). 

Downloaded the latest version of your 6.2.3 goldendict dictionary from the releases page (version 22.2alpha.a32ebfa2) and taking into account what you suggested about changing the path installation, this time I unzipped the goldendict files on C:\ (I've always put it in my external hard drives, among with all the dictionaries). When called, the program open instantly, but when a dictionary path is linked it happens as always. It crashes.

Is there any possibility of importing my dicts instead of adding one by one? (remember, there's a huge ~25Gbs of data.

Also:
>when running in console ,does it give any error message?

No, not at all.

P.S.: Is there any difference between versions?  There is this 5.15.2 and 6.2.3 versions for different platforms when a new release is published

**Comment by xiaoyifang** [comment]
> There is this 5.15.2 and 6.2.3 versions

these are qt versions

**Comment by darlopvil** [comment]
> > There is this 5.15.2 and 6.2.3 versions
> 
> these are qt versions

Tried both, same issue :(

**Comment by xiaoyifang** [comment]
> Is there any possibility of importing my dicts instead of adding one by one? (remember, there's a huge ~25Gbs of data.

portable mode.

**Comment by darlopvil** [comment]
> > Is there any possibility of importing my dicts instead of adding one by one? (remember, there's a huge ~25Gbs of data.
> 
> portable mode?

all versions are portable mode, aren't they? there's no .exe setup

**Comment by xiaoyifang** [comment]
create a `portable` folder and `content` folder alongside with executable file.then the index will be put into the content/index folder
.  by this way,you can move your index around without the need to build index again.

**Comment by xiaoyifang** [comment]
https://www.meziantou.net/tip-automatically-create-a-crash-dump-file-on-error.htm

follow the above instruction  to create a dump file and upload it here .

**Comment by darlopvil** [comment]
Hello there,

It was over a year since my last message but i'm here again with the same issue. This time I'm using a new computer for this and the problem, still remains. I'm using the last WinterSolstice.exe file from the githubreleases (no pre-release/alpha version). Same OS, Win10 x64 bits).

The problem:
GD suddenly crashes when importing/consulting dictionaries

What I've already done:
Changed the path. Made a new folder called "content" in the GD installation path. The opened GD, Reescaned, it crashes.

I've found some dumo files in my \AppData\Local\CrashDumps\ folder. Here you have the last one.
https://letsupload.cc/F7yfsdT4y6/GoldenDict.exe.10476_rar
